src/Model/Troop.php

```diff
@@ -67,4 +73,13 @@ public function getName(): string
     {
         return $this->name;
     }
+
+    public function setTiredness(): void
+    {
+        $this->tiredness = self::TIREDNESS_MAX;
+    }
```

Comment:
The name of the method should be resetTiredness()

Reply:
Modification ok